integration_tests/utils.js

```diff
@@ -162,7 +162,7 @@ const cleanupStackTrace = (output: string) => {
     .replace(/\n.*at.*assert\.js.*$/gm, '')
     .replace(/\n.*at.*node\.js.*$/gm, '')
     .replace(/\n.*at.*next_tick\.js.*$/gm, '')
-    .replace(/\n.*at Promise \(<anonymous>\).*$/gm, '')
+    .replace(/\n.*at (new)? Promise \(<anonymous>\).*$/gm, '')
```

Comment:
shouldn't this be `(new )?` (space after `new`)? Obviously it works, so maybe not :P

Reply:
You're right, didn't notice it broke older node 8 on circle 🤦‍♂️